neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -146,8 +147,63 @@ def get_experiment(self, experiment_id):
         fake_experiment = NonCallableMagicMock()
         # `timeOfCreation` is required by `TimeOffsetGenerator`
         fake_experiment.timeOfCreation = experiment.creationTime
+
+        try:
+            tags = self._alpha_backend.get_string_set_attribute(
```

Comment:
It is already returned in `self._get_attributes(experiment_id)` below so you don't need to do this api call.

Reply:
Indeed...
![](https://media.giphy.com/media/XeLcgh8gT8o0F5SQ8i/giphy.gif)